From 64a33f1beb4afc4b1b382fdadd6ac79a837a5e6f Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.org>
Date: Tue, 6 Feb 2018 11:48:17 +0000
Subject: [PATCH 194/454] BCM270X_DT: Minor cosmetic DT tidy

Deleting the timer node for all bcm27* DTBs than re-adding an identical
one for bcm2708 is wrong - just delete it where it isn't wanted.

Also change a #include to match the style of similar includes.

Signed-off-by: Phil Elwell <phil@raspberrypi.org>
---
 arch/arm/boot/dts/bcm2708.dtsi | 9 ---------
 arch/arm/boot/dts/bcm2709.dtsi | 2 ++
 arch/arm/boot/dts/bcm270x.dtsi | 4 +---
 arch/arm/boot/dts/bcm2710.dtsi | 2 ++
 4 files changed, 5 insertions(+), 12 deletions(-)

--- a/arch/arm/boot/dts/bcm2708.dtsi
+++ b/arch/arm/boot/dts/bcm2708.dtsi
@@ -3,15 +3,6 @@
 #include "bcm2708-rpi.dtsi"
 
 / {
-	soc {
-		timer@7e003000 {
-			compatible = "brcm,bcm2835-system-timer";
-			reg = <0x7e003000 0x1000>;
-			interrupts = <1 0>, <1 1>, <1 2>, <1 3>;
-			clock-frequency = <1000000>;
-		};
-	};
-
 	/delete-node/ cpus;
 
 	__overrides__ {
--- a/arch/arm/boot/dts/bcm2709.dtsi
+++ b/arch/arm/boot/dts/bcm2709.dtsi
@@ -6,6 +6,8 @@
 	soc {
 		ranges = <0x7e000000 0x3f000000 0x01000000>,
 		         <0x40000000 0x40000000 0x00040000>;
+
+		/delete-node/ timer@7e003000;
 	};
 
 	__overrides__ {
--- a/arch/arm/boot/dts/bcm270x.dtsi
+++ b/arch/arm/boot/dts/bcm270x.dtsi
@@ -1,5 +1,5 @@
 /* Downstream bcm283x.dtsi diff */
-#include "dt-bindings/power/raspberrypi-power.h"
+#include <dt-bindings/power/raspberrypi-power.h>
 
 / {
 	chosen {
@@ -8,8 +8,6 @@
 
 	soc: soc {
 
-		/delete-node/ timer@7e003000;
-
 		watchdog: watchdog@7e100000 {
 			/* Add alias */
 		};
--- a/arch/arm/boot/dts/bcm2710.dtsi
+++ b/arch/arm/boot/dts/bcm2710.dtsi
@@ -16,6 +16,8 @@
 			interrupt-parent = <&local_intc>;
 			interrupts = <9>;
 		};
+
+		/delete-node/ timer@7e003000;
 	};
 
 	__overrides__ {
